_posts/2020-04-26-fireside-chats.md

```diff
@@ -0,0 +1,18 @@
+---
+author: Mandy
+title: Fireside chats - winning hackathons and meeting an ER doctor
+---
+
+We have been having some awesome talks done by these experts recently:
+* Q and A with doctor/developer/designer Dr. Vincent Dumouchel
+* "How to Win a Hackathon" by Ben Zittlau
+
+## Q and A with doctor/developer/designer
+Dr. Vincent Dumouchel shared a lot with us about his career path through schooling, bootcamps and how he now has a part time gig as an ER doctor and as director of a tech agency. We talked about the difference about being on an ER team (think commanding and getting things done fast) and a tech team (think good communication and talking things through). We talked about one major need that most Canadian hospitals have is the need to monitor patients in waiting rooms because people can come in with a headache but then soon the symptoms of COVID-19 emerge and go from okay to ICU quite fast. We also talked about what a good leader is, whether that is at a company or on a hackathon team. Vincent said **"a leader is someone who makes space for everyone to talk. You need a leader on a hackathon team to organize everything and make sure everyone’s voice is heard"**. After the session was over we all commented on how warm it was to be able to talk to people about what we are going through. Vincent said it was important to **"take time to speak about your feelings"**
```

Comment:
....path through school, bootcamps, part time gig as an ER doctor, and director of a tech agency. We talked about the difference "between".....

Reply:
You: "We talked about one major need that most Canadian hospitals have is the need to monitor patients in waiting rooms because people can come in with a headache but then soon the symptoms of COVID-19 emerge and go from okay to ICU quite fast."  Me:  Canadian hospitals have an urgent need to monitor patients in waiting rooms, because people can walk in and appear ok, but rapidly go downhill and end up in ICU.

Reply:
I applied a similar re-flow